src/templates/items/parts/item-duration.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="form-group input-select">
+    <label>{{localize "SFRPG.EffectDurationTypesLabel"}}</label>
```

Comment:
It would also be neat to include a checkbox for whether the effect's icon should be shown on the actor's token. See `token.toggleEffect`